### PR TITLE
Adds dependabot for dependency management

### PR DESCRIPTION
**Pull request by esacteksab**

### CODE REVIEW

**Comment by dynamike** [comment]
@suzubara This PR will add dependabot to the repo, which submit PRs with updates the various JS modules. Right now it will run weekly.

File: .dependabot/config.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+version: 1
+update_configs:
+  # Keep go modules up to date, batching pull requests weekly
```
Comment (suzubara) [inline .dependabot/config.yml]: seems like this should be `js modules` :)

Comment (dynamike) [reply .dependabot/config.yml]: doh :)

Comment (esacteksab) [reply .dependabot/config.yml]: Ooops! My copy and paste is showing!

**Pull request merged**